Sources/OpenCombine/Utilities/Utilities.swift

```diff
@@ -0,0 +1,10 @@
+//
+//  Utilities.swift
+//
+//
+//  Created by Eric Patey on 17.08.2019.
+//
+
+/// A namespace for utility types.
+///
+public enum Utilities {}
```

Comment:
We cannot introduce new public APIs to maintain source compatibilty with Combine, can you please change the access level to internal?